src/resources/stepTemplates.ts

```diff
@@ -46,3 +46,48 @@ export const BaseStepsTemplate = [
     component: ViewComponentNames.REVIEW_CONFIRM
   }
 ]
+
+/** The step templates for COOPs. */
+export const CoopStepsTemplate = [
+  {
+    id: 'step-1-btn',
+    step: 1,
+    icon: 'mdi-domain',
+    text: 'Define Your\nCooperative',
+    to: RouteNames.DEFINE_COMPANY,
+    component: ViewComponentNames.DEFINE_COMPANY
+  },
+  {
+    id: 'step-2-btn',
+    step: 2,
+    icon: 'mdi-account-multiple-plus',
+    text: 'Add First\nDirectors',
+    to: RouteNames.ADD_PEOPLE_AND_ROLES,
+    component: ViewComponentNames.ADD_PEOPLE_AND_ROLES
+
+  },
+  {
+    id: 'step-3-btn',
+    step: 3,
+    icon: 'mdi-format-list-text',
+    text: 'Upload Rules',
+    to: RouteNames.CREATE_RULES,
+    component: ViewComponentNames.CREATE_RULES
```

Comment:
Hey, new route and component here, cool!

Reply:
We're growing!